src/applications/disability-benefits/526EZ/config/form.js

```diff
@@ -103,6 +104,16 @@ const treatments = ((treatmentsCommonDef) => {
 
 })(treatmentsSchema);
 
+const initialData = ((prefill) => {
```

Comment:
This seems more complicated than it needs to be, no? Why the immediately invoked function?

Reply:
:man_shrugging: